src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -179,7 +179,7 @@ const useCloudEnvironmentPolling = googleProject => {
   const load = async maybeStale => {
     try {
       const [newDisks, newRuntimes] = googleProject ? await Promise.all([
-        Ajax(signal).Disks.list({ googleProject, creator: getUser().email, includeLabels: 'saturnApplication' }),
+        Ajax(signal).Disks.list({ googleProject, creator: getUser().email, includeLabels: 'saturnApplication,saturnWorkspaceNamespace,saturnWorkspaceName' }),
```

Comment:
Are these actually needed here? Environments.js does its own GET.

Reply:
No. Thanks for the catch!